### ISSUE TITLE
Docs Navigation is confusing

### ISSUE THREAD

**Issue by natefinch** [juju/docs#272]
Someone on IRC asked about how to get started writing a charm, so I pointed them to the docs here: https://jujucharms.com/docs/authors-intro

However, I noticed the same thing I had trouble with when I started reading the docs - it's not at all clear how to navigate the site from this page.

By default, all sections of the navigation menu on the left are open, and so all the charm stuff is actually scrolled off bottom of the screen... and the actual contents of the page don't give you a way to navigate to the next page.  Finally, there's no indication about which page you're on in the navigation menu, and the name of the link doesn't always match up well with the url.

**Comment by evilnick** [comment]
Hi, I started a long reply to this, but basically, all the nav functionality is coded into jujucharms.com, it doesn't get it from the docs. So, problems with not indicating the current page, expanding the wrong sections etc need to be addressed there.

As for Previous/Next, we have resisted that as most of the pages are self contained topics with relevant links in the text, it is not necessarily a sequential thing. There are 2 links in the 3rd paragraph that lead to suggested further places to read, either with a practical exploration of charm writing, or an overview of the components. I will have a go at rewording that though.
